src/main/java/duke/EventCommand.java

```diff
@@ -0,0 +1,27 @@
+package duke;
+
+import java.time.LocalDateTime;
+
+/**
+ * Duke command for creating an Event and adding it
+ * to current tasks.
+ */
+public class EventCommand extends Command {
+    private String descriptor;
+
+    private LocalDateTime dateTime;
+
+    public EventCommand(String descriptor, LocalDateTime dateTime) {
+        this.descriptor = descriptor;
+        this.dateTime = dateTime;
+    }
+
```

Comment:
Perhaps the use of `@Override` annotation can be consistent across all the files too? [Google](https://google.github.io/styleguide/javaguide.html#s5-naming) recommends us to always use `@Override`.